src/python/pants/core/goals/style.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
The file `goals/style.py` suggests we're adding a goal `./pants style`. Could we rename the file to something like `style_util.py` or `style_request.py` please?